linux_sql/README.md

```diff
@@ -0,0 +1 @@
+testing github
```

Comment:
don't forget to update README file
PR name is wrong please review this https://www.notion.so/jarviscanada/Git-Github-GitFlow-Guidelines-4254192fe01e41bbbb2cce3f85185cda#1471b130c2cd44f1b49ae4f99eadd744